File: app/src/main/java/com/prashant/masterbuddy/ws/model/VideoItem.java
package com.prashant.masterbuddy.ws.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by tanmay.agnihotri on 5/19/18.
 */

public class VideoItem {

    @SerializedName("ID")
    private int id;

    @SerializedName("Title")
    private String title;

    @SerializedName("FileName")
    private String filename;

    @SerializedName("UploadedFile")
    private String uploadedFile;

    @SerializedName("Description")
    private String description;

    @SerializedName("UploadedThumbnailImage")
    private String uploadedThumbnailImage;

    @SerializedName("FullThumbnailImagePath")
    private String fullThumbnailImagePath;

    @SerializedName("UserID")
    private int userId;

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getFilename() {
        return filename;
    }

    public String getUploadedFile() {
        return uploadedFile;
    }

    public String getDescription() {
        return description;
    }

    public int getUserId() {
        return userId;
    }

    public String getUploadedThumbnailImage() {
        return uploadedThumbnailImage;
    }

    public String getFullThumbnailImagePath() {
        return fullThumbnailImagePath;
    }
}

File: app/src/main/java/com/prashant/masterbuddy/ws/model/VideoItem.kt
package com.prashant.masterbuddy.ws.model

import com.google.gson.annotations.SerializedName

/**
 * Created by tanmay.agnihotri on 5/19/18.
 */

class VideoItem {

    @SerializedName("ID")
    val id: Int = 0

    @SerializedName("Title")
    val title: String? = null

    @SerializedName("FileName")
    val filename: String? = null

    @SerializedName("UploadedFile")
    val uploadedFile: String? = null

    @SerializedName("Description")
    val description: String? = null

    @SerializedName("UploadedThumbnailImage")
    val uploadedThumbnailImage: String? = null

    @SerializedName("FullThumbnailImagePath")
    val fullThumbnailImagePath: String? = null

    @SerializedName("UserID")
    val userId: Int = 0
}
